jaq-core/src/error.rs

```diff
@@ -56,6 +56,15 @@ pub enum Error {
     Custom(String),
 }
 
+impl Error {
+    /// Convert an error into a Val, to be used for `catch`
+    /// expressions.
+    pub fn as_val(&self) -> Val {
+        // TODO: make it better, Jude
+        Val::Null
+    }
+}
+
```

Comment:
You should not need this function; just use to_string().

Reply:
"[The movement you need is on your shoulder](https://genius.com/983074/The-beatles-hey-jude/The-movement-you-need-is-on-your-shoulder)" -- damn those Beatles!

Reply:
What about the `Error::Val` case? I think we need this to work and yield `true`:` `try error(1) catch . + 1 == 2`. Anyway, I see that even then the `match` can simply be moved to the parser; I'll do that for now.

Reply:
A potential test: `try (0|.[]) catch .`.